features/steps/user_initialisation_steps.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=E0102, E0602
+from behave import *
+from selenium.webdriver.support.ui import Select
+
+from features.fixtures import get_chrome_browser
+from features.steps.helpers import sign_in
+from users.factories import AdminUserFactory
+from users.factories import UserFactory
+
+
+@given("an admin exists in database")
+def step_impl(context):
+    context.admin = AdminUserFactory()
+
+
+@when("the user signs up")
+def step_impl(context):
+    # TODO: HAVE TO SOLVE CAPTCHA AND EMAIL VERIFICATION ISSUES
```

Comment:
Haven't you already solve captcha issue? Does not `CAPTCHA_TEST_MODE` setting work?

Reply:
It does, I forgot to change this comment. I guess it should be more like: `Write proper implementation reflecting user's behaviour` or something like that. However this most likely won't get merged until I do that.
Otherwise I'm requesting changes myself! 😛

Reply:
Done, the step got implemented